README.md

```diff
@@ -116,7 +116,7 @@ You can use the Taipy Studio extension in Visual Studio Code to configure your s
 Your configuration is automatically saved as a TOML file.<br />
 Check out Taipy Studio [Documentation](https://docs.taipy.io/en/latest/manuals/studio/)
 
-For more advanced use cases or if you prefer coding your configurations instead of using Taipy Studio,<br />
+For more advanced use cases or if you want to make your own configurations instead of using Taipy Studio,<br />
```

Comment:
There is a difference between "make configuration" and "coding configuration". With Taipy Studio, you can do configuration without coding.